src/quo2/components/selectors/styles.cljs

```diff
@@ -1,130 +1,175 @@
 (ns quo2.components.selectors.styles
```

Comment:
if possible could you also adjust name space to best practices..
i.e 
`quo2.components.selectors.selectors.view` and quo2.components.selectors.selectors.style`

Reply:
I renamed them! now they look better.
499fee8bfb0762733e67bcd0bf519548d805bef6